launcher/ui/pages/modplatform/flame/FlamePage.cpp

```diff
@@ -341,7 +341,7 @@ void FlamePage::setSearchTerm(QString term)
 
 void FlamePage::createFilterWidget()
 {
-    auto widget = ModFilterWidget::create(nullptr, false, this);
+    auto widget = ModFilterWidget::create(nullptr, false, ModPlatform::ResourceProvider::FLAME, this);
```

Comment:
Remove the extended argument(as you already pass the modplatform)
`extended= provider == ModPlatform::ResourceProvider::MODRINTH;`
see:https://github.com/PrismLauncher/PrismLauncher/pull/3504/files#diff-fb990c568c0adc2841953c01dd033d4db8b8fb4321e4b6d63937f204af11b674R124

Reply:
Ah, so that's what `extended` means. Thanks for the clarification